packages/server/test/course.integration.ts

```diff
@@ -626,21 +626,53 @@ describe('Course Integration', () => {
         },
       ];
 
+      // Counts total professor courses before registration
+      const totalProfCoursesBefore = await UserCourseModel.count({
+        where: { userId: professor.id },
+      });
+
       await supertest({ userId: professor.id })
         .post(`/courses/register_courses`)
         .send(registerCourses)
         .expect(201);
 
+      // total professor courses after registering 2 courses
+      const totalProfCourses = await UserCourseModel.count({
+        where: { userId: professor.id },
+      });
+      expect(totalProfCourses).toEqual(totalProfCoursesBefore + 2);
+
+      // verify courses are created as expected
       const ubw = await CourseModel.findOne({
         sectionGroupName: 'Underwater Basket-Weaving',
       });
       const ubw2 = await CourseModel.findOne({
         sectionGroupName: 'Underwater Basket-Weaving 2',
       });
-      const totalProfCourses = await UserCourseModel.count();
-      expect(totalProfCourses).toEqual(2);
 
-      // checks if the course has the professor as a userCourse
+      expect(ubw.name).toEqual('Scuba');
```

Comment:
Hmm I think we can assume that the validating the contents of the course are covered by other tests, right? I believe Iris meant just checking the new objects we created in the endpoint -- I will add some new lines to another commit.